kubetest/main.go

```diff
@@ -480,6 +480,10 @@ func prepareGcp(o *options) error {
 			return fmt.Errorf("fail to set project %s : err %v", p, err)
 		}
 
+		if err := os.Setenv("PROJECT", p); err != nil {
```

Comment:
I mentioned https://github.com/kubernetes/test-infra/pull/3528#issuecomment-315915283, so the plan is we are going to clean up those 1mil scripts?